### ISSUE TITLE
nan for train_loss and val_loss during training

### ISSUE THREAD

**Issue by laure0406** [MIC-DKFZ/nnUNet#2636]
Hello everyone !

I am currently trying to use nnUNet to segment some MRIs. I have 260 of them, with their asociated labels.
When I run the training, with nnUNetv2_train 100 3d_fullres 0 -tr nnUNetTrainer_250epochs , (after running the preprocess), from the 1st epoch, i have train_loss nan and val_loss nan.
I don't know where the problem comes from. I checked the documentation my files seem in order (json file, organisation and name of folder...) 
If someone could help me I would be very grateful !!
Don't hesitate to ask for more information if you need some to understand my issue.

Thank you so much,
Have a nice day !!
![error](https://github.com/user-attachments/assets/97443a7d-05fc-44e3-a173-e917ef37ce13)

PS: here is what i used:

----------- Pre-process ----------
nnUNetv2_plan_and_preprocess -d 100 -c 3d_fullres --verify_dataset_integrity -np 2

----------- Training ----------
nnUNetv2_train 100 3d_fullres 0 -tr nnUNetTrainer_250epochs

**Comment by Lars-Kraemer** [comment]
Hey @laure0406 

Nans in the loss often has something to do with the data.
Therefore as a first step, I’d recommend checking your data. Are there any corrupted images or masks? Are all classes present in both the training and validation sets?
Let me know if you found anything there; if not, we'll keep looking.

Best,
Lars
